sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamUncollectRelTest.java

```diff
@@ -44,4 +57,56 @@ public void testNodeStats() {
     Assert.assertEquals(4d, estimate.getWindow(), 0.001);
     Assert.assertEquals(0., estimate.getRate(), 0.001);
   }
+
+  @Test
+  public void testUncollectPrimitive() {
+    registerTable(
+        "PRIMITIVE",
+        TestBoundedTable.of(
+                Schema.FieldType.STRING,
+                "user_id",
+                Schema.FieldType.array(Schema.FieldType.INT32),
+                "ints")
+            .addRows("1", Arrays.asList(1, 2, 3)));
+
+    String sql = "SELECT * FROM unnest(SELECT ints from PRIMITIVE)";
```

Comment:
Could we use `SELECT * FROM unnest([1, 2, 3])` as the test query?

Reply:
yup thats better thank you